frontend/src/pages/Login.jsx

```diff
@@ -1,14 +1,44 @@
 import React, { useState } from "react";
 import "./style/LoginSignup.scss";
 import IdentificationHeader from "../components/IdentificationHeader";
+import axios from "axios";
+import { useHistory } from "react-router-dom";
+import { connect } from "react-redux";
 
-export default function Login() {
+function Login(props) {
+  let history = useHistory();
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+
+  const sublogin = e => {
+    e.preventDefault();
+    axios
+      .post("http://localhost:5000/auth/login", {
+        mail: email,
+        password
+      })
+      .then(({ data }) => {
+        console.log(data);
+        //TODO redirect to "/"
```

Comment:
Do not add TODOs: either you do it, or you add it to the backlog, but that's not the place to note this